wordpress-mobile/WordPress-iOS
# ULS: Remove unifiedSiteAddress flag.

Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/476

This removes the `unifiedSiteAddress` feature flag. In Auth, the master `unifiedAuth` flag is used to show unified Site Address.

To test:

Verify these paths show the unified Site Address flow:

![site_address](https://user-images.githubusercontent.com/1816888/93832621-80a43c00-fc33-11ea-9773-589bd629d84f.png)

1. Prologue > `Enter your site address`.
2. Login Epilogue > `Connect another site`.
3. Social login failure > retry with site address. To trigger this failure:
  - Continue with Google.
  - Select a Google account that has WP 2FA configured via SMS.
  - On the 2FA view, go back and select the same Google account.
  - On the error view, select `Try with the site address`.

![social_error](https://user-images.githubusercontent.com/1816888/93832562-59e60580-fc33-11ea-8ab4-ad6bb77b99a6.png)


PR submission checklist:
